apps/files_trashbin/lib/AppInfo/Application.php

```diff
@@ -55,20 +62,18 @@ public function register(IRegistrationContext $context): void {
 		$context->registerServiceAlias('principalBackend', Principal::class);
 
 		$context->registerUserMigrator(TrashbinMigrator::class);
+
+		$context->registerEventListener(NodeWrittenEvent::class, EventListener::class);
+		$context->registerEventListener(BeforeUserDeletedEvent::class, EventListener::class);
+
+		// pre and post-rename, disable trash logic for the copy+unlink case
+		$context->registerEventListener(BeforeNodeDeletedEvent::class, Trashbin::class);
+		$context->registerEventListener(BeforeFileSystemSetupEvent::class, EventListener::class);
```

Comment:
```suggestion

		$context->registerEventListener(NodeWrittenEvent::class, EventListener::class);
		$context->registerEventListener(BeforeUserDeletedEvent::class, EventListener::class);
		$context->registerEventListener(BeforeFileSystemSetupEvent::class, EventListener::class);

		// pre and post-rename, disable trash logic for the copy+unlink case
		$context->registerEventListener(BeforeNodeDeletedEvent::class, Trashbin::class);
```
Please reorder to make it clearer the comment is only about BeforeNodeDeletedEvent, and to group by listening class.